docs/reviews/2026-04-15/20-30-concurrency.md

```
@@ -0,0 +1,93 @@
+**[P2] VillageScene.ts — base64url 디코딩 실패로 자기 아바타 중복**
+- 파일: `frontend/src/game/scenes/VillageScene.ts:382-387`
+- `atob()`는 base64url 인코딩을 지원하지 않는다. guest JWT subject가 `guest-<uuid>`이므로 `-`/`_` 문자가 포함될 수 있고, 이 경우 `atob()` 예외 발생 → `myDisplayId = null` → 자신의 broadcast를 필터링하지 못해 아바타가 중복으로 렌더링된다.
+- 권고: base64url → base64 변환 함수 적용 또는 `jose` 라이브러리의 JWT 디코더 사용.
+
+---
+
+## 보완 검증 (Grep/Read 기반)
+
+### [CRITICAL]
+
+#### CONC-C1. @Transactional 내 외부 LLM API 호출
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)